### PR TITLE
Use correct function arity

### PR DESCRIPTION
**Pull request by ooflorent**
**What kind of change does this PR introduce?**

refactoring

**Did you add tests for your changes?**

no

**If relevant, link to documentation update:**

n/a

**Summary**

Pass the correct number of arguments on plugins.

**Does this PR introduce a breaking change?**

no

**Other information**

### CODE REVIEW

File: lib/dependencies/HarmonyImportSpecifierDependency.js [hunk 1]
```diff
@@ -131,7 +131,6 @@ class HarmonyImportSpecifierDependency extends HarmonyImportDependency {
 
 HarmonyImportSpecifierDependency.Template = class HarmonyImportSpecifierDependencyTemplate extends HarmonyImportDependency.Template {
 	apply(dep, source, runtime) {
-		super.apply(dep, source, runtime);
```
Comment (ooflorent) [inline lib/dependencies/HarmonyImportSpecifierDependency.js]: This call was no-op

Comment (sokra) [reply lib/dependencies/HarmonyImportSpecifierDependency.js]: You should still keep it. Maybe we add code to `HarmonyImportDependencyTemplate.apply` in future.

Comment (ooflorent) [reply lib/dependencies/HarmonyImportSpecifierDependency.js]: Sure, I'll revert this

**Comment by sokra** [comment]
Thanks

**Pull request merged**